lms/static/js/discovery/result_item_view.js

```diff
@@ -0,0 +1,50 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext',
+    'date'
+], function ($, _, Backbone, gettext, Date) {
+    'use strict';
+
+    function formatDate(date) {
+        return dateUTC(date).toString('MMM dd, yyyy');
```

Comment:
Doing the formatting client-side is going to make this difficult to localize the date format.  Is there any way we could send the localized date string from the server instead of formatting it here?  (There are utility functions on the server that already handle date localization.)